bin/configurable-http-proxy

```diff
@@ -107,3 +109,17 @@ log.info("Proxy API at %s://%s:%s/api/routes",
     (listen.api_ip || '*'),
     listen.api_port);
 
+if (options.redirectPort) {
+  var http = require('http');
+  
+  http.createServer(function (req, res) {
+    var host = req.headers['host'].split(':')[0];
+    
+    // Make sure that when we redirect, it's to the port the proxy is running on
+    if (listen.port !== 443 && listen.port !== 80) {
```

Comment:
Zero sense whatsoever. The only reason I put this in here was a check for non-standard ports. If they're trying to use HTTPS on port 80, I'm currently shuddering. Perhaps some additional warning messages here?